starter-code/index.js

```diff
@@ -1,11 +1,37 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort((x, y) => x - y);
+    this.length++;
+  }
+  get(pos) {
+    return this.items[pos-1];
+  }
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList")
+    }
+    else return this.items[this.length-1];
```

Comment:
El punto y coma ya no es necesario, no lo uses. gracias x'D
```suggestion
    else return this.items[this.length-1]
```